modules/linear_boltzmann_solvers/discrete_ordinates_curvilinear_problem/discrete_ordinates_curvilinear_problem.cc

```diff
@@ -357,7 +357,10 @@ DiscreteOrdinatesCurvilinearProblem::ComputeSecondaryUnitIntegrals()
 }
 
 std::shared_ptr<SweepChunk>
-DiscreteOrdinatesCurvilinearProblem::SetSweepChunk(LBSGroupset& groupset)
+DiscreteOrdinatesCurvilinearProblem::SetSweepChunk(LBSGroupset& groupset,
+                                                   size_t max_level_size,
```

Comment:
I think I intended these parameters to be temporary. It might be best to add these as members of `LBSProblem`.